packages/forklift-console-plugin/src/modules/Providers/ProviderRow.tsx

```diff
@@ -60,49 +49,52 @@ const fromProviderState = (status: ProviderStatus): 'Error' | 'Ok' | 'Unknown' |
   }
 };
 
-const toNegativeState = (conditionValue: string): 'Error' | 'Ok' | 'Unknown' => {
-  switch (conditionValue) {
-    case 'True':
-      return 'Error';
-    case 'False':
-      return 'Ok';
-    default:
-      return 'Unknown';
-  }
+const CategoryToIconMap = {
+  Critical: <RedExclamationCircleIcon />,
+  Error: <RedExclamationCircleIcon />,
+  Required: <GreenCheckCircleIcon />,
+  Warn: <BlueInfoCircleIcon />,
+  Advisory: <GreenCheckCircleIcon />,
 };
 
-const StatusCell = ({
-  value,
-  entity: { positiveConditions, negativeConditions, phase },
-  t,
-}: CellProps) => {
-  const allConditions = [
-    [positiveConditions, toPositiveState],
-    [negativeConditions, toNegativeState],
-  ].flatMap(
-    ([conditions, toState]: [
-      SupportedConditions,
-      (value: string) => 'Error' | 'Ok' | 'Unknown',
-    ]) => [
-      ...Object.values(conditions)
-        .filter(Boolean)
-        .map(({ message, status }) => {
-          return <StatusIcon key={message} status={toState(status)} label={message} />;
-        }),
-    ],
-  );
-
-  const label = PROVIDER_STATUS[value]?.(t) ?? t('Unknown');
-  return (
-    <Popover
-      hasAutoWidth
-      bodyContent={<div>{allConditions.length > 0 ? allConditions : t('No information')}</div>}
+const StatusCell = ({ value, entity: { phase, object }, t }: CellProps) => {
+  const allConditions = (object?.status?.conditions || []).map(({ message, category }) => (
+    <Flex
+      key={message.valueOf()}
+      spaceItems={{ default: 'spaceItemsXs' }}
+      display={{ default: 'inlineFlex' }}
+      flexWrap={{ default: 'nowrap' }}
     >
-      <Button variant="link" isInline aria-label={label}>
-        <StatusIcon status={fromProviderState(phase)} label={label} />
-      </Button>
-    </Popover>
-  );
+      <FlexItem>{CategoryToIconMap[category] || <BlueInfoCircleIcon />}</FlexItem>
+      <FlexItem>{message}</FlexItem>
+    </Flex>
+  ));
+
+  const CellLabel = PROVIDER_STATUS[value]?.(t) ?? t('Unknown');
```

Comment:
I'd prefer `cellLabel` - starting with capital letter suggests that this is a React component.